packages/react-components/src/components/Picker/Picker.tsx

```diff
@@ -35,7 +35,7 @@ export const Picker: React.FC<IPickerProps> = ({
   onOpen,
   onClose,
   openedOnInit = false,
-  clearSearchAfterSelection,
+  clearSearchAfterSelection = true,
```

Comment:
IMO we shouldn't treat that change as a fix. 🤔 It's a breaking change, changing how the search works on every component implementation that does not use this prop.